assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -43,6 +43,7 @@ export default function UserInputSelectOptions( {
 	max,
 	next,
 	isActive,
+	noInstructions,
```

Comment:
```suggestion
	noInstructions,
```

I suspect the reason this was used is the slightly nicer API of `<UserInputSelectOptions noInstructions />` over `<UserInputSelectOptions instructions={ false } />`. But the double-negative, especially in the code below:

```js
			{ ! noInstructions && (
				<p className="googlesitekit-user-input__select-instruction">
					<span>
						{ sprintf(
							/* translators: %s: number of answers allowed. */
							_n(
								'Select only %d answer',
								'Select up to %d answers',
								max,
								'google-site-kit'
							),
							max
						) }
					</span>
				</p>
			) }
```

is harder to parse than:

```js
			{ showInstructions && (
				<p className="googlesitekit-user-input__select-instruction">
					<span>
						{ sprintf(
							/* translators: %s: number of answers allowed. */
							_n(
								'Select only %d answer',
								'Select up to %d answers',
								max,
								'google-site-kit'
							),
							max
						) }
					</span>
				</p>
			) }
```

Or maybe even `hideInstructions` would be a better variable name, as it's a bit more of an instructive prop name to be used than setting a negatively-named boolean to `true`. It's just a bit roundabout to me. 😅

Reply:
Hey @tofumatt, thanks for the comment. I was, in part, taking my cue from other components, for example:

https://github.com/google/site-kit-wp/blob/af97300672d201e874227729b02ac009ae20791c/assets/js/components/user-input/UserInputPreview.js#L113-L133

However, I agree the double negative is not so nice to read, and am happy to change it to `showInstructions`.

Reply:
Ah fair point. The struggle between consistency and partial improvements 😅

Thanks, I think this is all good now 👍🏻